PoESkillTree.Engine.GameModel/PassiveTree/Base/JsonPassiveTreeConstants.cs

```diff
@@ -29,9 +31,37 @@ public class JsonPassiveTreeConstants
         public int PSSCentreInnerRadius { get; set; } = 130;
 
         [JsonProperty("skillsPerOrbit")]
-        public float[] SkillsPerOrbit { get; private set; } = new float[] { 1f, 6f, 12f, 12f, 40f };
+        public int[] SkillsPerOrbit { get; private set; } = new int[] { 1, 6, 12, 12, 40 };
```

Comment:
If I understood https://github.com/grindinggear/skilltree-export/commit/8b80ed1da2698e8e414ae657daaaf09c21240615#diff-b335630551682c19a781afebcf4d07bf978fb1f8ac04c6bf87428ed5106870f5 correctly (the "Nodes (New)" column in the first table), shouldn't this be 1, 6, 16, 16, 40 now?

Reply:
Sorry, I didn't see this for some reason. 

This is just the default if `skillsPerOrbit` isn't supplied in `constants`. I think they started providing them in 3.6, but I could be wrong. The 3.17.0 has: ```        "skillsPerOrbit": [
            1,
            6,
            16,
            16,
            40,
            72,
            72
        ]```

Reply:
Ah, ok then. No worries.